### PR TITLE
Issue 1532 Update Text for Create DAC form

### PR DESCRIPTION
**Pull request by horus2021**
#1532 Revise Text for Create DAC form

### CODE REVIEW

File: src/components/views/EditDAC.jsx [hunk 1]
```diff
@@ -259,10 +259,10 @@ class EditDAC extends Component {
                     <Input
                       name="title"
                       id="title-input"
-                      label="Community cause"
+                      label="Name your Community"
                       type="text"
                       value={dac.title}
-                      placeholder="e.g. Hurricane relief."
```
Comment (aminlatifi) [inline src/components/views/EditDAC.jsx:265]: This should not be changed

File: src/components/views/EditDAC.jsx [hunk 2]
```diff
@@ -259,10 +259,10 @@ class EditDAC extends Component {
...
+                      placeholder="Describe your altruistic community (DAC) in 1 sentence"
                       help="Describe your Decentralized Altruistic Community (DAC) in 1 sentence."
```
Comment (aminlatifi) [inline src/components/views/EditDAC.jsx:266]: change attribute to `helpText` and set the value "Describe your altruistic community (DAC) in 1 sentence"

Comment (aminlatifi) [inline src/components/views/EditDAC.jsx:266]: It's the one which should be changed

Comment (horus2021) [reply src/components/views/EditDAC.jsx]: @aminlatifi  You are right see new commit.

**Pull request merged**